res/controllers/Denon-Prime-4-scripts.js

```diff
@@ -0,0 +1,869 @@
+            on: colDeck[i],
+        });
+        Prime4.assignmentButtons[i].trigger();
+    }
+
+    Prime4.mixerA = new mixerStrip(1, 0);
+    Prime4.mixerB = new mixerStrip(2, 1);
+    Prime4.mixerC = new mixerStrip(3, 2);
+    Prime4.mixerD = new mixerStrip(4, 3);
+
+    // Initialize effect banks
+    Prime4.effectBank = [];
+    for (let i = 0; i <= 1; i++) {
+        Prime4.effectBank[i] = new components.EffectUnit([i + 1, i + 3]);
+        for (let j = 0; j < 3; j++) {
+            Prime4.effectBank[i].enableButtons[j + 1].midi = [0x96 + i, 0x06 + j];
+            Prime4.effectBank[i].knobs[j + 1].midi = [0xB6 + i, 0x01 + j];
+            Prime4.effectBank[i].knobs[j + 1].input = Prime4.EffectUnitEncoderInput;
+        }
+        Prime4.effectBank[i].dryWetKnob.midi = [0xB6 + i, 0x04];
+        Prime4.effectBank[i].dryWetKnob.input = Prime4.EffectUnitEncoderInput;
+        Prime4.effectBank[i].effectFocusButton.midi = [0x96 + i, 0x0A];
+        Prime4.effectBank[i].init();
+    }
+
+    // Load song to deck with library encoder button
+    Prime4.encoderLoad = new components.Button({
+        midi: [0x9F, 0x06],
+        group: "[Library]",
+        key: "GoToItem",
+    });
+
+    // View Button
+    Prime4.maxView = new components.Button({
+        midi: [0x9F, 0x07],
+        group: "[Master]",
+        key: "maximize_library",
+        type: components.Button.prototype.types.toggle,
+    });
+
+    // BACK Button
+    Prime4.moveBack = new components.Button({
+        midi: [0x9F, 0x03],
+        group: "[Library]",
+        key: "MoveFocusBackward",
+    });
+
+    // FWD Button
+    Prime4.moveForward = new components.Button({
+        midi: [0x9F, 0x04],
+        group: "[Library]",
+        key: "MoveFocusForward",
+    });
+
+    // Sweep FX - Filter Button
+    Prime4.sweepFilter = new components.Button({
+        midi: [0x9F, 0x0C],
+        group: "[QuickEffectRack1_[Channel1]]",
+        key: "enabled",
+        on: 0x02,
+        off: 0x01,
+        input: function(_channel, _control, value, _status, _group) {
+            for (let i = 1; i <= 4; i++) {
+                if (value > 0) {
+                    const effect = "[QuickEffectRack1_[Channel" + i + "]]";
+                    if (engine.getParameter(effect, "enabled") > 0) {
+                        engine.setParameter(effect, "enabled", 0);
+                    } else {
+                        engine.setParameter(effect, "enabled", 1);
+                    }
+                }
+            }
+        },
+    });
+
+    // Sweep FX - Echo Button
+    Prime4.sweepEcho = new components.Button({
+        midi: [0x9F, 0x0D],
+    });
+    // Sweep FX - Wash Button
+    Prime4.sweepWash = new components.Button({
+        midi: [0x9F, 0x0E],
+    });
+    // Sweep FX - Noise Button
+    Prime4.sweepNoise = new components.Button({
+        midi: [0x9F, 0x0F],
+    });
+
+    Prime4.leftDeck.reconnectComponents();
+    Prime4.rightDeck.reconnectComponents();
```

Comment:
if you comment both of these out, do the decks still have LED output? Do they still clash?